task_5.py

```diff
@@ -1,6 +1,16 @@
 def factorial(n):
-    pass
-
+    silnia = 1
+    for i in range(n,n +1):
+        if n==0 or n==1:
+            return 1
```

Comment:
Tego ifa możesz w całości wyciągnąć przed pętle `for`. Dzięki temu nie będzie musiał go sprawdzać przy każdym obrocie pętli.